dotnet/src/SemanticKernel.Skills/Skills.OpenAPI/OpenApi/OpenApiDocumentParser.cs

```diff
@@ -186,6 +216,71 @@ private static IList<RestApiOperationPayloadProperty> GetPayloadProperties(strin
         return result;
     }
 
+    /// <summary>
+    /// Returns parameter value.
+    /// </summary>
+    /// <param name="name">The parameter name.</param>
+    /// <param name="valueMetadata">The value metadata.</param>
+    /// <returns>The parameter value.</returns>
+    private static string? GetParameterValue(string name, IOpenApiAny valueMetadata)
+    {
+        var value = valueMetadata as IOpenApiPrimitive;
+        if (value == null)
+        {
+            return null;
+        }
+
+        switch (value.PrimitiveType)
+        {
+            case PrimitiveType.Integer:
+                var intValue = (OpenApiInteger)(IOpenApiPrimitive)value;
```

Comment:
Do we need to cast it to `IOpenApiPrimitive` again here, if we already did it at the beginning of the method?

Reply:
No, no need in the cast. Fixed.